src/features/session_replay/aggregate/index.js

```diff
@@ -56,94 +55,93 @@ export class Aggregate extends AggregateBase {
 
     handle(SUPPORTABILITY_METRIC_CHANNEL, ['Config/SessionReplay/Enabled'], undefined, FEATURE_NAMES.metrics, this.ee)
 
-    const shouldSetup = hasDependentSettings(agentIdentifier)
-
-    if (shouldSetup) {
-      // The SessionEntity class can emit a message indicating the session was cleared and reset (expiry, inactivity). This feature must abort and never resume if that occurs.
-      this.ee.on(SESSION_EVENTS.RESET, () => {
-        this.scheduler.runHarvest()
-        this.abort(ABORT_REASONS.RESET)
-      })
-
-      // The SessionEntity class can emit a message indicating the session was paused (visibility change). This feature must stop recording if that occurs.
-      this.ee.on(SESSION_EVENTS.PAUSE, () => { this.recorder?.stopRecording() })
-      // The SessionEntity class can emit a message indicating the session was resumed (visibility change). This feature must start running again (if already running) if that occurs.
-      this.ee.on(SESSION_EVENTS.RESUME, () => {
-        if (!this.recorder) return
-        // if the mode changed on a different tab, it needs to update this instance to match
-        const { session } = getRuntime(this.agentIdentifier)
-        this.mode = session.state.sessionReplayMode
-        if (!this.initialized || this.mode === MODE.OFF) return
-        this.recorder?.startRecording()
-      })
+    // The SessionEntity class can emit a message indicating the session was cleared and reset (expiry, inactivity). This feature must abort and never resume if that occurs.
+    this.ee.on(SESSION_EVENTS.RESET, () => {
+      this.scheduler.runHarvest()
```

Comment:
Just noticing some things that have been uncaught that could be safe guarded.  `this.scheduler` might not exist here.